main.py

```diff
@@ -52,11 +52,13 @@ def main(data_file, break_time):
         strategy.order()
         buy, sell = strategy.get_order()
 
-        # Put all the information in a 'bitcoin_price.csv' file
-        csv_file.append_data_file(price, date, ema_long, ema_short, macd, rsi, adx)
-
-        # Put all the information in a postgres data base
-        postgres.transfer_data(price, date, ema_long, ema_short, macd, rsi, adx)
+        # Put all teh information to csv file or postgres data base
+        if data_file.startswith("postgres://"):
```

Comment:
i think we discussed about better name. Not 
`@click.option('--data_file', help = 'Path to datafile.')` but

`@click.option('--data_source', help = 'Data source for persisting data. Can be either path to csv or connection string to postgres database')`